src/lib/jsonParse/jsonQueryContextRequest.cpp

```diff
@@ -240,7 +240,7 @@ static std::string scopeValue(const std::string& path, const std::string& value,
 
     reqDataP->qcr.scopeP->value = value;
 
-    reqDataP->qcr.scopeP->stringFilterP = new StringFilter();
+    reqDataP->qcr.scopeP->stringFilterP = new StringFilter(SftQ);
```

Comment:
I have seen a couple of cases previously:

```
new StringFilter()
```

now

```
new StringFilter(SftQ)
```

Could you elaborate a little bit on this new parameter in StringFilter constrution, please? (maybe at skype)